www/base/src/app/builders/build/build.controller.coffee

```diff
@@ -31,42 +32,55 @@ class Build extends Controller
                         icon: "spinner fa-spin"
                 ]
 
-        buildbotService.bindHierarchy($scope, $stateParams, ['builders', 'builds'])
-        .then ([builder, build]) ->
-            if not build.number? and buildnumber > 1
-                $state.go('build', builder:builderid, build:buildnumber - 1)
-            breadcrumb = [
-                    caption: "Builders"
-                    sref: "builders"
-                ,
-                    caption: builder.name
-                    sref: "builder({builder:#{builderid}})"
-                ,
-                    caption: build.number
-                    sref: "build({build:#{buildnumber}})"
-            ]
+        opened = dataService.open($scope)
```

Comment:
With my limited English skills I couldn't find a better one. I am open to anything else.